test/e2e/operator_test.go

```diff
@@ -6,53 +6,40 @@ import (
 	"context"
 	"fmt"
 	"net"
-	"net/http"
 	"os"
 	"testing"
 	"time"
 
-	awsSession "github.com/aws/aws-sdk-go/aws/session"
-	configv1 "github.com/openshift/api/config/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	kscheme "k8s.io/client-go/kubernetes/scheme"
-
-	operatorv1alpha1 "github.com/openshift/external-dns-operator/api/v1alpha1"
-
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/wait"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
 
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
-	"github.com/aws/aws-sdk-go/service/route53"
+	operatorv1alpha1 "github.com/openshift/external-dns-operator/api/v1alpha1"
 )
 
 const (
-	recordExistsTimeout  = 5 * time.Minute
-	podReadyTimeout      = 1 * time.Minute
-	httpGetTimeout       = 10 * time.Minute
-	recordDeletedTimeout = 5 * time.Minute
-	dnsLookupTimeout     = 5 * time.Second
-	dnsServer            = "8.8.8.8:53"
+	hostedZoneDomain   = "example-test.info"
```

Comment:
The hosted zone need to be randomized so concurrent executions of the e2e tests do not interfere with each other.